scaffolding-chef/plan.ps1

```diff
@@ -1,7 +1,7 @@
 $pkg_name="scaffolding-chef"
 $pkg_description="Scaffolding for Chef Policyfiles"
 $pkg_origin="chef"
-$pkg_version="0.6.0"
+$pkg_version="0.5.0"
```

Comment:
Should the version for the scaffolding be the same for windows and Linux?

Reply:
We still need to do a version `0.5.0` release for Windows, then that will get updated to 0.6.0 later.

Reply:
See: #20

Reply:
sounds good